plugins/modules/inventory_intent.py

```diff
@@ -2886,10 +2965,10 @@ def verify_diff_deleted(self, config):
 
         if device_delete_flag:
             self.status = "success"
-            self.msg = "Requested Devices - {0} Deleted from Cisco DNA Center and Deletion verified.".format(str(input_devices))
-            self.log(self.msg)
+            self.msg = "Requested Devices '{0}' Deleted from Cisco Catalyst Center and Deletion verified.".format(str(input_devices))
+            self.log(self.msg, "INFO")
         else:
-            self.log("Playbook paramater doesnot match with the Cisco DNA Center means Device Deletion task not executed successfully.")
+            self.log("Playbook paramater doesnot match with the Cisco Catalyst Center means Device Deletion task not executed successfully.", "INFO")
```

Comment:
self.log("Mismatch between playbook parameter and Cisco Catalyst Center detected, indicating that the device deletion task may not have executed successfully.", "INFO")
Also you can specify what parameter ?

Reply:
added the device ip for which exists after deletion operation in cisco catalyst center.